tests/hlsl-intrinsic/quad-control/quad-control-comp-functionality.slang

```diff
@@ -1,14 +1,18 @@
 //TEST(compute):COMPARE_COMPUTE_EX:-vk -compute -shaderobj -emit-spirv-directly
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -profile cs_6_7 -dx12 -use-dxil -shaderobj -render-feature hardware-device
-//TEST(compute):COMPARE_COMPUTE_EX:-metal -compute -shaderobj 
+//TEST(compute):COMPARE_COMPUTE_EX:-metal -compute -shaderobj -DMETAL
```

Comment:
Is -D supported on COMPARE_COMPUTE_EX tests?

You may want to write `-xslang -DMETAL` instead.